### DISCUSSION TITLE
Estimation of time needed for TI-AB screening

### DISCUSSION THREAD

**Discussion by Hanedinges** [asreview/asreview#1470] [Q&A]
Dear all, 

I wonder: how can we estimate the time (left) needed for ti-ab screening? 

My team has proposed as a stopping rule: 100 irrelevant articles and estimated it would take 20 hours to reach to that point. How do I know how realistic this is?

And what to think of this: in a recent [publication](https://psycharchives.org/en/item/cb2c6618-0acf-4a5a-976d-23f1d54ecd8e) in which Rayyan and ASReview are being compared, it is adviced not to rely on one criterion but use 'combinations to improve performance  E.g. screen min 50% and stop after 50 consecutive irrelevants' 

50%!!! Hmm, I thought, let's estimate how much time it would take to screen another 22% (we've screened almost 3%) See table below:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/simon/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/simon/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Date | % Articles screened | Number of articles   screened | Minutes | Tempo minutes/article | Time (h) for another   22% van 10000
-- | -- | -- | -- | -- | --
14-6-2023 | 97,40% | 260 | 487 | 2 | 73

</body>

</html>

According to this estimation it would take another 73 hours to screen another 22% and we do not know if the other stopping rule will be applicable by then (how can I formulate this correctly??). 

That is a huge difference with 20 hours. 
I realise in fact I have two questions: how can we estimate the time needed in case we use a simple single stopping rule of 100 irrelevant articles?
And what to think of the two criterion advice and the estimation I made based on a scenario in which we use two? 

Hopefully there's a rule of thumb? I would be grateful if someone could help me get a clearer picture. 

Cheers! Simone

**Comment by j0sien** [comment]
You could check out the 4-step heuristic that Rens and I came up with,.
It is not published yet but you could look at this picture:
![afbeelding](https://github.com/asreview/asreview/assets/63192964/1e2fc7e6-7b71-4297-908c-6453b898627a)
Boetje, Josien (2023). Graphical overview of the SAFE procedure for applying a practical stopping heuristic for active learning-aided systematic reviewing.. figshare. Figure. https://doi.org/10.6084/m9.figshare.22227199.v1

Or the poster we made with it:
Boetje, Josien (2023). Poster explaining the SAFE Procedure. figshare. Poster. https://doi.org/10.6084/m9.figshare.23522733.v1

**Reply by Rensvandeschoot** [reply]
see also the pre-print "The SAFE Procedure: A Practical Stopping Heuristic for Active Learning-Based Screening in Systematic Reviews and Meta-Analyses": https://psyarxiv.com/c93gq
